FeedSimulator/src/main/scala/bythebay/FeederActor.scala

```diff
@@ -0,0 +1,53 @@
+package bythebay
+
+import akka.actor.{Props, Actor, ActorLogging}
+import bythebay.FeederActor.Initialize
```

Comment:
(minor nit) I think it slightly better to just `import FeederActor._` in the body of the FeederActor class.